lte/gateway/fabfile.py

```diff
@@ -219,7 +219,8 @@ def s1ap_setup_cloud():
 
 def integ_test(
     gateway_host=None, test_host=None, trf_host=None,
-    destroy_vm='True', provision_vm='True',
+    destroy_vm='True', build=None, hash=None,
```

Comment:
you added hash here but you're getting from elsewhere on line 548?

Reply:
yeah I was about to write the feature that @uri200 requested but wanted to ship that quickly, will get back to it in a different PR